lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.cron.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+/**
+ * @file
+ * Provides cron job/drush command for dosomething_campaign_run.module.
+ */
+
+/**
+ * Implements hook_cron().
+ */
+function dosomething_campaign_run_cron() {
+  // Get all campaign runs and the start and end date for each translation of the run date field.
+  $results = db_query("SELECT campaign_run.nid as campaign_run_nid,
+                              run_date.field_run_date_value as start_date,
+                              run_date.field_run_date_value2 as end_date,
+                              run_date.language as language,
+                              campaigns.field_campaigns_target_id as campaign_nid
+                       FROM node as campaign_run
```

Comment:
this will be more clear once we have runs created for this stuff and you could see the thor content :smiling_imp: :sena: